demo/full/scripts/controllers/Main.jsx

```diff
@@ -1,4 +1,4 @@
-import RxPlayer from "rx-player";
+import RxPlayer from "../../../../src/index.ts";
```

Comment:
esbuild doesn't do alias directly. It relies on `jsconfig.json` and `tsconfig.json` files that can define it for it.

However I did not want to add yet another config file at the root for what is moreover only a demo need.
So I went back to relative imports, which are exactly the same thing.